Mathlib/MeasureTheory/Function/ContinuousMapDense.lean

```diff
@@ -324,33 +324,26 @@ theorem Integrable.exists_boundedContinuous_integral_sub_le [μ.WeaklyRegular] {
 
 namespace Lp
 
-variable (E)
+variable (E μ)
 
 /-- A function in `Lp` can be approximated in `Lp` by continuous functions. -/
-theorem boundedContinuousFunction_dense [SecondCountableTopologyEither α E] [_i : Fact (1 ≤ p)]
-    (hp : p ≠ ∞) [μ.WeaklyRegular] : (boundedContinuousFunction E p μ).topologicalClosure = ⊤ := by
-  rw [AddSubgroup.eq_top_iff']
-  intro f
-  refine mem_closure_iff_frequently.mpr ?_
-  rw [Metric.nhds_basis_closedBall.frequently_iff]
-  intro ε hε
-  have A : ENNReal.ofReal ε ≠ 0 := by simp only [Ne, ENNReal.ofReal_eq_zero, not_le, hε]
+theorem boundedContinuousFunction_dense [SecondCountableTopologyEither α E] [Fact (1 ≤ p)]
+    (hp : p ≠ ∞) [μ.WeaklyRegular] :
+    Dense (boundedContinuousFunction E p μ : Set (Lp E p μ)) := fun f ↦ by
+  refine (mem_closure_iff_nhds_basis EMetric.nhds_basis_closed_eball).2 fun ε hε ↦ ?_
```

Comment:
```suggestion
    Dense (boundedContinuousFunction E p μ : Set (Lp E p μ)) := by
  intro f  
  refine (mem_closure_iff_nhds_basis EMetric.nhds_basis_closed_eball).2 fun ε hε ↦ ?_
```
It feels weird to mix term mode and tactic mode when most of the proof is tactic mode. If you really want to golf, you can put the `f` in the `refine` line, but I don't really see the point.